SafeNeighbourhood/Pages/Index.cshtml

```diff
@@ -19,19 +19,20 @@
             <th>Street_Number</th>
             <th>Property ID</th>
         </tr>
-        @{
```

Comment:
I removed this annotation. Since code inside is annotated.

Reply:
Did you test this without the annotation?

Reply:
Yes I did. It worked perfectly fine.